WTG.Analyzers/Rules/Rules.xml

```diff
@@ -95,6 +95,11 @@
 			<message>Prefer an if-throw over assigning a coalesce-throw expression to the discard symbol.</message>
 			<description>Prefer an if-throw over assigning a coalesce-throw expression to the discard symbol.</description>
 		</rule>
+		<rule id="17" name="LocalCouldBeConfusedWithDiscard" severity="Info">
+			<title>Don't define locals that could be confused with discards.</title>
```

Comment:
I'm not sure I like calling it a "local" here. A local _what_?

Reply:
'local' is a common abbreviation of 'local variable'. though since this now covers parameters too, should probably replace 'local' with 'variable'.